src/permutation.rs

```diff
@@ -22,7 +22,7 @@ pub struct Permutation {
 impl Default for Permutation {
     fn default() -> Self {
         let width = 9;
-        let full_founds = 4;
+        let full_founds = 8;
```

Comment:
To avoid confusion, the code can call this `total_full_rounds`

Reply:
Seems the best, it's done! Thanks for catching it.

Reply:
The permutation field `full_rounds` would also need to be changed, as it is referring to the `total_full_rounds`

Reply:
Done!